packages/patternfly-4/react-table/src/components/Table/utils/formatters/inlineEditFormatterFactory.js

```diff
@@ -0,0 +1,26 @@
+const inlineEditFormatterFactory = ({ renderEdit, renderValue, resolveValue, isEditable = null } = {}) => (
+  value,
+  additionalData
+) => {
+  const { rowData, columnIndex } = additionalData;
+
+  let result;
```

Comment:
No need to assign to result if we are only returning it. You can add return instead of assign to each if statements and it wil be the same.

Reply:
you are right, thanks